paper-input-container.html

```diff
@@ -277,6 +279,20 @@
         @apply(--paper-input-container-input);
       }
 
+	  .input-content ::content input:focus,
+      .input-content ::content textarea:focus,
+      .input-content ::content iron-autogrow-textarea:focus,
+      .input-content ::content .paper-input-input:focus {
+        @apply(--paper-input-container-input-focus);
+      }
+
+	  .input-content ::content input[invalid],
```

Comment:
I think this should be `.input-content.is-invalid ::content input` and friends (these friends, in particular: https://github.com/PolymerElements/paper-input/blob/master/paper-input-container.html#L259) , like we use it here: https://github.com/PolymerElements/paper-input/blob/master/paper-input-container.html#L249